src/main/java/com/netgrif/application/engine/importer/service/FieldFactory.java

```diff
@@ -136,8 +136,9 @@ Field getField(Data data, Importer importer) throws IllegalArgumentException, Mi
             field.setFormat(format);
         }
         if (data.getView() != null) {
+            log.warn("Data attribute [view] is deprecated.");
```

Comment:
it might be helpful to log which data field has the view tag, so that it can be replaced more easily

Reply:
done